README.md

```diff
@@ -75,4 +75,8 @@ Note: There is no UI validation for the input, as the class inspector will not b
 
 ### Require Confirmation
 
-If the `require_confirm_worker_names` option is configured with the worker class name of the job you want to run, it will prompt for confirmation. By clicking `OK`, the job will be scheduled to run; otherwise, it will not run. This helps to prevent accidentally running a critical job when not meant to be.
+If you're using sidekiq adhoc jobs in Production, you may want to consider using this configuration as an extra protection against erroneously running a sidekiq job. Once turned on, the user will be required to enter a challenge keyword in a prompt before the job can be run through the admin UI.
+
+To use this feature, add the worker class name of the jobs you would like to add the confirmation for into the `require_confirm_worker_names` option.
+
+The challenge keyword is set to be `confirm` as default, if you would like to configure the challenge keyword, you can use the `require_confirm_prompt_message` option.
```

Comment:
@haile-kaligo One last fix for the README. Could you add the option description to the `Options` list above under `Usage` section? Then all should be good.

Reply:
Got it 👍